### DISCUSSION TITLE
facebook group migrator?

### DISCUSSION THREAD

**Discussion by ctranter** [theabbie/mewe#5] [Ideas]
hi,
Thanks for this project. I wonder would it be possible to create a facebook group post importer? I'm an ex developer now an architect so I'm a bit rusty on the programming :) plus I was .net based. I am involved in a number of large groups on facebook and some of us are trying to get people over to MeWe. The majority of group members don't want to move because "everything is here". If I could replicate the group posts over that would be a real big bonus to get people to follow. I am going to try myself to cobble something together using this code but it'll take a while :)
Interested in your thoughts?

**Comment by theabbie** [comment]
@ctranter That's a great idea, Will try to figure out.

**Reply by ctranter** [reply]
great stuff. I have checked the facebook api and it seems to work as you'd need it to.
https://developers.facebook.com/docs/groups-api/

my idea was to have a service that read the group posts and then used your API to post to mewe.
